warehouse/models/staging/stg_waste_emission.sql

```diff
@@ -0,0 +1,227 @@
+{% set cnd_split = 0.5 %}
+{% set residual = 0.22 %}
+{% set hard_split = 0.1 %}
+
+WITH ef AS (
+    SELECT
+        material,
+        recycled,
+        landfilled,
+        combusted,
+        composted,
+        CASE
+            WHEN material = 'Yard Trimmings' THEN 5
+            WHEN material = 'Mixed Organics' THEN 4
+            WHEN material = 'Food Waste' THEN 3
+            WHEN material = 'Mixed Recyclables' THEN 2
+            ELSE 1
+        END AS group_id
+    FROM {{ source("raw", "waste_emission_factors_EPA") }}
+    WHERE
+        material IN (
+            'Mixed MSW', 'Yard Trimmings', 'Food Waste', 'Mixed Recyclables', 'Mixed Organics'
+        )
+),
+
+roll AS (
```

Comment:
this selects data from the final_waste_recycle reference, assigns a group_id based on the type of material. The group_id is used to categorize different types of waste materials